.github/workflows/social-media-automation.yml

```
@@ -1,6 +1,11 @@
 name: Social Media Automation
 
 on:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)